x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityTests.java

```diff
@@ -514,6 +529,81 @@ public void testLicenseUpdateFailureHandlerUpdate() throws Exception {
         }
     }
 
+    public void testSecurityHandlerIsAlwaysInstalled() throws IllegalAccessException {
+        final Logger amLogger = LogManager.getLogger(ActionModule.class);
+        Loggers.setLevel(amLogger, Level.DEBUG);
+        final MockLogAppender appender = new MockLogAppender();
+        Loggers.addAppender(amLogger, appender);
+        appender.start();
+
+        Settings settings = Settings.builder()
+            .put("xpack.security.enabled", false)
+            .put("path.home", createTempDir())
+            .build();
+        SettingsModule settingsModule = new SettingsModule(Settings.EMPTY);
+        ThreadPool threadPool = new TestThreadPool(getTestName());
+
+        try {
+            UsageService usageService = new UsageService();
+            Security security = new Security(settings, null);
+
+            appender.addExpectation(new MockLogAppender.SeenEventExpectation(
+                "Security rest wrapper", ActionModule.class.getName(), Level.DEBUG,
+                "Using REST wrapper from plugin org.elasticsearch.xpack.security.Security"
+            ));
+
+            ActionModule actionModule = new ActionModule(settingsModule.getSettings(),
+                TestIndexNameExpressionResolver.newInstance(threadPool.getThreadContext()),
+                settingsModule.getIndexScopedSettings(), settingsModule.getClusterSettings(), settingsModule.getSettingsFilter(),
+                threadPool, Arrays.asList(security), null, null, usageService, null);
+            actionModule.initRestHandlers(null);
```

Comment:
This call is no longer needed since all necessary logic is within ActionModule's constructor.